astropy/coordinates/solar_system.py

```diff
@@ -43,79 +43,140 @@
                                        ('neptune', [(0, 8)]),
                                        ('pluto', [(0, 9)]))
                                       )
-SOLAR_SYSTEM_BODIES = tuple(BODY_NAME_TO_KERNEL_SPEC.keys())
 
-_JPL_EPHEM_NOTE = """
-    This calculation uses JPL Ephemeris SPK filesto calculate the body's
-    location, defaulting to DE430.  It will be downloaded the first time this
-    function is used and cached from then on.  To change this, set
-    ``kernel_url`` as described in the coordinate documentation.
+"""Indices to the plan94 routine for the given object."""
+PLAN94_BODY_NAME_TO_PLANET_INDEX = OrderedDict(
+    (('mercury', 1),
+     ('venus', 2),
+     ('earth-moon-barycenter', 3),
+     ('mars', 4),
+     ('jupiter', 5),
+     ('saturn', 6),
+     ('uranus', 7),
+     ('neptune', 8)))
+
+_EPHEMERIS_NOTE = """
+You can either give an explicit ephemeris or use a default, which is normally
+an approximate ephemeris that does not require ephemeris files.  To change
+the default to be a JPL ephemeris::
+
+    >>> from astropy import coordinates as coord
```

Comment:
for this small set of examples, maybe drop this and just spell out the full `coordinates`?